ui/components/Header.js

```diff
@@ -211,20 +210,23 @@ function K8sContextMenu({
         </div>
       </IconButton>
 
-      <Slide direction="down" timeout={300} in={open} style={{ position : "absolute", left : "-5rem", zIndex : "-1", bottom : "-75%", transform : showFullContextMenu ? `translateY(${transformProperty}%)`: "translateY(0)" }} mountOnEnter unmountOnExit>
+      <Slide direction="down"  timeout={400} in={open} style={{ backgroundColor : "#EEEEEE", position : "absolute", left : "-5rem", zIndex : "-1", bottom : "-100%", transform : showFullContextMenu ? `translateY(${transformProperty}%)`: "translateY(0)" }} mountOnEnter unmountOnExit>
```

Comment:
@Harshit-Dandriyal Could you please remove the inline styles here and add it through using makeStyle?